### ISSUE TITLE
Survey Response Doesn't Appear to POST Map Data

### ISSUE THREAD

**Issue by mzagaja** [MAPC/myschoolcommute2#127]
When sending a new survey response from the main survey interface it does not appear to POST data from the map as evidenced by the following log:

```
Started GET "/assets/react_modules/intersectingstreets.min.self-bba5a8edcce5cd9ca466e8f7b717ad706f7fd2c8d18d737b5e65634f2f7676a0.js?body=1" for 127.0.0.1 at 2017-08-03 15:51:10 -0400
Started POST "/survey_responses" for 127.0.0.1 at 2017-08-03 15:51:27 -0400
Processing by SurveyResponsesController#create as HTML
Parameters: {"utf8"=>"✓", "authenticity_token"=>"sEASjO/k3Izb4owVoMQ/3Np885kxsfYDg4yTu/UQ+RzHPZbRgPh1fVjyIZG30+Ai+6PhpLfH33tJ8UTlRIsJNQ==", "-search"=>"", "survey_response"=>{"grade_1"=>"k", "to_school_1"=>"w", "from_school_1"=>"w", "nr_vehicles"=>"1", "nr_licenses"=>"1", "survey_id"=>"2"}, "commit"=>"Submit Survey"}
```

**Comment by allthesignals** [comment]
Related to #130, these are the same issue

**Comment by mzagaja** [comment]
@allthesignals Cool, when I was trying to debug before I left I did notice that your keyboard update seemed to fix it POSTing.

**Comment by mzagaja** [comment]
Appears to be fixed in #132
